fac29/NisaInvest-TFB-FE
# Connect contact form

## Description
Added Formspree to contact form, enabling functionality to email Fahan directly from contact form.

## Related Issue
Closes #77 

## How Can This Be Tested?
Fill in the contact form on `http://localhost:5173/contact`. A success message and option to refresh the form to send a new enquiry should appear.

The email will appear in Formspree. Login details are already shared.

## Checklist:
